android-gpgs/src/de/golfgl/gdxgamesvcs/GpgsPlayerData.java

```diff
@@ -0,0 +1,42 @@
+package de.golfgl.gdxgamesvcs;
+
+import de.golfgl.gdxgamesvcs.player.IPlayerData;
+
+/**
+ * Gpgs Player
+ * <p>
+ * Created by Kari Vatjus-Anttila on 27.11.2021
+ */
+
+public class GpgsPlayerData implements IPlayerData {
```

Comment:
Since this class is basically a generic implementation if IPlayerData, I wouldn't mind to have it in core. That's not done for other classes, but I think it would be better when it had been done that way. :)